class.mesh-settings.php

```diff
@@ -329,12 +329,12 @@ static public function get_default_tab_slug() {
 	 * @return   array $tabs Settings tabs
 	 */
 	static public function get_tabs() {
-
-		$tabs                 = array();
-		$tabs['settings']  = __( 'Settings',   'mesh' );
-		$tabs['faq']       = __( 'About Mesh', 'mesh' );
-		$tabs['changelog'] = __( 'Change Log', 'mesh' );
-		$tabs['linchpin']  = __( 'About Linchpin', 'mesh' );
+		$tabs = array(
```

Comment:
I feel like @desrosj did this for a reason. I can't remember why at the moment. Gonna snoop into previous commits / pull requests.